extractor/src/main/java/com/example/extractor/ConfigSerializer.java

```diff
@@ -67,7 +68,11 @@ public static String serializeAllInOneConfigToString(String[] exampleFilePaths,
 
         // Read the template and replace the placeholder
         String template = Files.readString(Path.of(templateFilePath), StandardCharsets.UTF_8);
-        return TemplateProcessor.replacePlaceholders(template, combinedModuleContent, isTreeWalker);
+
+        String result = TemplateProcessor.replacePlaceholders(template, combinedModuleContent, isTreeWalker);
+
+        // Remove duplicate id properties
+        return removeDuplicateIdProperties(result);
```

Comment:
Please add logic to `private static Configuration addIdProperty(Configuration`

Reply:
I tried doing that, but it didn'w give the right config. It would remove example4 value and give the moduleName. Anyways, will try again. Can i add this logic of the method in addIdPorperty?

Reply:
Add logic to skip I'd in this method.
Or make method that check if it already exists in config, and skip `addIdProperty`

Reply:
@Romani i tried doing that but it is failing. Any issues with the current approach?

Reply:
Current is too hacky.
What is a problem to look in Configuration class for property with name " id" ?